documentation/data-importer.adoc

```diff
@@ -330,166 +270,86 @@ As mentioned before, if any element in your model is dashed, you will **not** be
-
-== Cypher statements
-
-You can also see the Cypher commands used to create the elements in your model.
-This can be helpful if you want to gain appreciation for the underlying Cypher required to execute the load.
-Let's have a look at _Order_ nodes.
-Click _Show Cypher_ on _Order_ which reveals the key statement:
-
-.Key statement
-[source,cypher]
-----
-CREATE CONSTRAINT `imp_uniq_Order_id` IF NOT EXISTS
-FOR (n: `Order`)
-REQUIRE (n.`id`) IS UNIQUE;
-----
-
-This ensures that the `orderID` is unique which is important since this is what is used later to create the relationship between the _Order_ nodes and the _Product_ nodes.
-As you will see later on, this constraint is also what allows you to run the import more than once.
-
-The load statement creates all the _Order_ nodes and sets all their properties.
-Both key and load statements look similar to this for the _Product_ nodes.
 
-But let's have a look at the Cypher statement to create the relationships:
+Congratulations on modelling, mapping, and loading your data! You can now explore and query it to learn more about the power of graphs and Neo4j. 
+We have a few very useful importer tips to complete this guide.
 
-.Load statement
-[source,cypher]
-----
-UNWIND $relRecords AS relRecord
-MATCH (source: `Order` { `orderID`: toInteger(trim(relRecord.`orderID`)) })
-MATCH (target: `Product` { `productID`: toInteger(trim(relRecord.`productID`)) })
-MERGE (source)-[r: `CONTAINS`]->(target);
-SET r.`unitPrice` = toFloat(trim(relRecord.`unitPrice`))
-SET r.`quantity` = toInteger(trim(relRecord.`quantity`))
-SET r.`discount` = toFloat(trim(relRecord.`discount`));
-----
-
-From this you can see how the source node and the target node are `MATCH` ed (found in the data) and then a `MERGE` statement is used to create a `CONTAINS` relationship _from_ the source node _to_ the target node.
-You can also see the properties added to each relationship.
-
-== Explore the graph
-
-With the data imported it's time to explore.
-Go ahead and navigate to the _Explore_ tab or click the button below:
-
-button::Explore[role=NX_TAB_NAV,tab=explore]
-
-The _Explore_ tab allows you to visualize and interact with your data with minimal use of Cypher.
-To start out, let's have a quick look at a sample of your data.
-
-You can see that the search bar in the top left corner is already populated with _Show me a graph_.
-Click the bar and press enter.
-As well as showing the sample graph, you can use the search bar to find and display any of the elements in your data.
-You can use near-natural language for this and that means that Explore understands a vocabulary based on the categories, labels, relationship types, property keys and property values of your dataset.
+[TIP]
+====
+Note that if you run this guide after the _Getting Started_, the count may show _updated_ rather than _created_ since Data Importer will ensure duplicate IDs are not created for elements already loaded.
```

Comment:
Missing ending  "guide" in "Note that if you run this guide after the _Getting Started_,"?